src/main/scala/mesosphere/marathon/core/task/update/TaskStatusUpdateProcessor.scala

```diff
@@ -5,5 +5,5 @@ import org.apache.mesos.Protos.TaskStatus
 import scala.concurrent.Future
 
 trait TaskStatusUpdateProcessor {
-  def publish(status: TaskStatus): Future[Unit]
+  def publish(status: TaskStatus, ack: Boolean = true): Future[Unit]
```

Comment:
I think a comment on why we decided to add an ack flag might clarify this as well